<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configurazione</class>
 <widget class="QMainWindow" name="Configurazione">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Configurazione</string>
  </property>
  <property name="windowIcon">
   <iconset resource="risorse.qrc">
    <normaloff>:/resources/icon_mmas.png</normaloff>:/resources/icon_mmas.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu {
  background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0.0181818 rgba(233, 240, 243, 255), stop:0.372727 rgba(247, 247, 247, 255), stop:0.659091 rgba(247, 247, 247, 255), stop:1 rgba(233, 240, 243, 255));
	font: 11pt &quot;Arial&quot;;
border: 1px outset rgb(188, 189, 207);
     border-radius: 1px;
 }

 QMenu::item {
     /* sets background of menu item. set this to something non-transparent
         if you want menu color and menu item color to be different */
     background-color: transparent;
font: 11pt &quot;Arial&quot;;
 }

 QMenu::item:selected { /* when user selects item using mouse or keyboard */
     background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 226, 229, 255), stop:0.254545 rgba(247, 247, 247, 255), stop:0.722727 rgba(247, 247, 247, 255), stop:1 rgba(220, 226, 229, 255));
color:black;
	font: 11pt &quot;Arial&quot;;
 }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">/* sets background color  */
QWidget {
background-color:rgba(255,255,255,0);
}
QWidget#centralWidget{
	/*background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(217, 239, 255, 255), stop:0.5 rgba(239, 248, 255, 255), stop:1 rgba(210, 236, 255, 255));
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(223, 246, 255, 255), stop:0.481132 rgba(243, 243, 243, 255), stop:0.995261 rgba(223, 246, 255, 255));*/
	
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(220, 232, 243, 255), stop:0.35545 rgba(247, 247, 247, 255), stop:0.635071 rgba(247, 247, 247, 255), stop:0.933649 rgba(220, 232, 243, 255));
	
}

QFrame{
	
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0.0181818 rgba(233, 240, 243, 255), stop:0.372727 rgba(247, 247, 247, 255), stop:0.659091 rgba(247, 247, 247, 255), stop:1 rgba(233, 240, 243, 255));
 border: 1px inset rgb(188, 189, 207);
     border-radius: 5px;
}

QToolTip {
     border: 1px solid rgb(188, 189, 207);
     padding: 0.1px;
    border-radius:1px;
	color: rgb(188, 189, 207);
	background-color: rgb(255, 255, 255);
   background-origin:padding-box;
     opacity: 100;
 }

QListWidget{	
border: 1px groove rgb(188, 198, 207);
}

/*table style*/
QTableView {
     border:2px groove rgb(188, 198, 207);   
	background-color: rgba(255, 255, 255,0);
	alternate-background-color:rgba(210, 233, 241,130);
	
 }

 QHeaderView::section {
/* background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(169, 228, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.00947867 rgba(220, 231, 240, 255), stop:1 rgba(239, 251, 254, 255));
     border: 0.5px groove rgb(188, 198, 207);
     height:20px;
	font: 11pt &quot;Arial&quot;;
 }

QTableView QTableCornerButton::section {
     background: rgb(255, 255, 255);
     border: 0px;
 }

QTabWidget::pane { /* The tab widget frame */
     border-top: 2px groove rgb(188, 198, 207);
 
 }

QSpinBox{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.208531 rgba(212, 223, 232, 255), stop:0.905213 rgba(239, 251, 254, 255));
}

QLabel{   
	border-bottom:1px solid;
     border-bottom-color: rgb(188, 198, 207);
    background-color: rgba(255, 255, 255, 0);
     border-radius: 5px;
     padding: 0 0px;
}

/* set scrollbar style */
QScrollBar:horizontal {
 border: 0px solid rgb(168, 172, 204);
 border-radius:7px;
	 background-color:rgba(255,255,255,0);
     height: 11px;
     margin: 1px 19px 0px 19px;
 }

 QScrollBar::handle:horizontal {	
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.0272727 rgba(146, 179, 203, 255), stop:1 rgba(228, 240, 248, 255));
     min-width: 11px;
   border: 1px solid rgb(168, 172, 204);
   border-radius:5px;
 }
 QScrollBar::add-line:horizontal {
  background-color:rgba(255,255,255,0);
   width: 13px;
   height:13px;
     subcontrol-position: right;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:horizontal {
  background-color:rgba(255,255,255,0);
     width: 13px;
    height:13px;
     subcontrol-position: left;
     subcontrol-origin: margin;
 }

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
	 background-color:rgba(255,255,255,0);
 }

  QScrollBar:left-arrow:horizontal{
	image: url(:/resources/back_not_hov.png);
   width:13px;
   height:13px;
     }
   QScrollBar:left-arrow:horizontal:hover{
	image: url(:/resources/back_not_hov.png);
   width:15px;
   height:15px;
     }
  QScrollBar:left-arrow:horizontal:pressed{
	image: url(:/resources/back_not_hov.png);
   width:11px;
   height:11px;
     }
 QScrollBar:right-arrow:horizontal{
	image: url(:/resources/next_not_hov.png);
   width:13px;
   height:13px;
     }
  QScrollBar:right-arrow:horizontal:hover{
	image: url(:/resources/next_not_hov.png);
   width:15px;
   height:15px;
     }
  QScrollBar:right-arrow:horizontal:pressed{
	image: url(:/resources/next_not_hov.png);
   width:11px;
   height:11px;
     }

QScrollBar:vertical {
 border: 0px solid rgb(188, 189, 207);
 border-radius:7px;
	background-color: rgba(255,255,255,0);
    width: 12px;
     margin: 19px 0px 19px 1px;
 }

 QScrollBar::handle:vertical {
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0.0272727 rgba(146, 179, 203, 255), stop:1 rgba(228, 240, 248, 255));
     min-height: 13px;
   border: 1px solid rgb(168, 172, 204);
   border-radius:5px;
 }
 QScrollBar::add-line:vertical {
 background-color:rgba(255,255,255,0);
   width: 13px;
   height:13px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     background-color:rgba(255,255,255,0);
     width: 13px;
    height:13px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
	background-color:rgba(255,255,255,0);
 }

  QScrollBar:up-arrow:vertical{
	image: url(:/resources/up_arrow.png);
   width:13px;
   height:13px;
     }
  QScrollBar:up-arrow:vertical:hover{
	image: url(:/resources/up_arrow.png);
   width:15px;
   height:15px;
     }
  QScrollBar:up-arrow:vertical:pressed{
	image: url(:/resources/up_arrow.png);
   width:11px;
   height:11px;
     }
 QScrollBar:down-arrow:vertical{
	image: url(:/resources/down_arrow.png);
   width:13px;
   height:13px;
     }
 QScrollBar:down-arrow:vertical:hover{
	image: url(:/resources/down_arrow.png);
   width:15px;
   height:15px;
     }
 QScrollBar:down-arrow:vertical:pressed{
	image: url(:/resources/down_arrow.png);
   width:11px;
   height:11px;
     }



</string>
   </property>
   <widget class="QPushButton" name="apply_conf">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>applica configurazione</string>
    </property>
    <property name="styleSheet">
     <string notr="true">/* pushbutton style */
QPushButton {
     border: 1px groove rgba(255,255,255,0);
     border-radius: 5px;
	background-color: rgba(255, 255, 255,0);
	
	image: url(:/resources/filter_ok_not_hov.png);


 }
 QPushButton:hover {
     border: 1px outset rgb(188, 189, 207);
	border-radius:3px;
	background-color: rgba(255, 255, 255, 0);
	image: url(:/resources/filter_ok.png);
 } 


QPushButton:pressed {
     border: 3px inset rgb(188, 189, 207);
	border-radius:3px;

 }
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_num">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="text">
        <string>Num. di elementi per classe</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spin_num">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>41</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>25</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="table_disp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="table_sel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>231</width>
      <height>401</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0.0181818 rgba(233, 240, 243, 255), stop:0.372727 rgba(247, 247, 247, 255), stop:0.659091 rgba(247, 247, 247, 255), stop:1 rgba(233, 240, 243, 255));
 border: 1px inset rgb(188, 189, 207);
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>CLASSI MARCHE DISPONIBILI</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>231</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0.0181818 rgba(233, 240, 243, 255), stop:0.372727 rgba(247, 247, 247, 255), stop:0.659091 rgba(247, 247, 247, 255), stop:1 rgba(233, 240, 243, 255));
 border: 1px inset rgb(188, 189, 207);
     border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>CLASSI MARCHE SELEZIONATE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="risorse.qrc"/>
 </resources>
 <connections/>
</ui>
